### ISSUE TITLE
Can rmarkdown make 508 compliant PDFs

### ISSUE THREAD

**Issue by ckelley-air** [rstudio/rmarkdown#1497]
Hi, 

Is there any way to increase the accessibility of pdfs created by r-markdown? For many projects pdfs need to meet 508 accessibility standards. For PDFs this means having "tags" that help screen readers parse the contents of the pdf. Is there any way to do something like this in R markdown?

I asked this on stack over flow (
https://stackoverflow.com/questions/51601975/can-i-add-pdf-tags-to-r-markdown-document)  and was pointed towards pandoc (which allowed me to create alt text for images, but not create the kind of tag structure that I need). Is there any way of doing this in Rmarkdown (or Sweave)? 

Thanks! 
ps. this is to some extent possible in ConTexT so i think its not outside the realm of possibility that it might be possible in rmarkdown.

**Comment by adunning** [comment]
This is a limitation of LaTeX: there's a [PDF Accessibility](https://www.tug.org/twg/accessibility/) page that outlines some of the attempts to support this: due to lack of resources nobody has yet developed a complete solution (although there is a [`tagpdf` package](https://www.ctan.org/pkg/tagpdf) in development, and is certainly the most promising effort so far).

**Comment by yihui** [comment]
I guess there isn't much we could do on rmarkdown's side since this is a pure LaTeX issue. Thanks for the report!
